src/components/SearchResults.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "../styles/search-results.css";
+
+function SearchResults({ results }) {
+  if (!results.length) {
+    return <p>No results</p>;
+  }
+  return (
+    <>
+      <h2> Results </h2>
+      <div className="search-results">
+        {results.map((result) => {
+          return (
+            <img
+              src={result}
+              alt="spaceImage"
+              className="search-results_image"
+            />
```

Comment:
In your browsers console you are getting an error because you haven't provided a unique `key` to this element. When you do a map in react, you should always provide a unique key that is relevant to item in the array but doesn't care about it's position (i.e. do not use it's index as the key). This improves performance. You can read more about this error [here](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key). In this case, because `result` is a unique string for each image, you can use this as the key, i.e.
```
{results.map((result) => {
  return (
    <img
      src={result}
      alt="spaceImage"
      className="search-results_image"
      key={result}
    />
  );
})}
```